### DISCUSSION TITLE
Biome v1.2 Rule promotions

### DISCUSSION THREAD

**Discussion by Conaclos** [biomejs/biome#273] [RFC]
Biome has a special category named nursery. All rules of this category are unstable and experimental rules.
I propose to promote the following rules to a stable category because no bug was reported since the last minor release of Biome. 

### A11y

- [noAriaUnsupportedElements](https://www.biomejs.dev/linter/rules/no-aria-unsupported-elements)
- [noNoninteractiveTabindex](https://www.biomejs.dev/linter/rules/no-noninteractive-tabindex)
- [noRedundantRoles](https://www.biomejs.dev/linter/rules/no-redundant-roles)
- [useAriaPropTypes](https://www.biomejs.dev/linter/rules/use-aria-prop-types)
  I propose renaming the rule to `useValidAriaValues`

### Complexity

- [noBannedTypes](https://www.biomejs.dev/linter/rules/no-banned-types)
- [noStaticOnlyClass](https://www.biomejs.dev/linter/rules/no-static-only-class)
- [noUselessEmptyExport](https://www.biomejs.dev/linter/rules/no-useless-empty-export)
- [noUselessThisAlias](https://www.biomejs.dev/linter/rules/no-useless-this-alias)

### Correctness

- [noConstantCondition](https://www.biomejs.dev/linter/rules/no-constant-condition)
- [noNonoctalDecimalEscape](https://www.biomejs.dev/linter/rules/no-nonoctal-decimal-escape)
- [noSelfAssign](https://www.biomejs.dev/linter/rules/no-self-assign)

### Style

- [useLiteralEnumMembers](https://www.biomejs.dev/linter/rules/use-literal-enum-members)
- [useNamingConvention](https://www.biomejs.dev/linter/rules/use-naming-convention)

### Suspicious

- [noControlCharactersInRegex](https://www.biomejs.dev/linter/rules/no-control-characters-in-regex)
- [noUnsafeDeclarationMerging](https://www.biomejs.dev/linter/rules/no-unsafe-declaration-merging)
- [useGetterReturn](https://www.biomejs.dev/linter/rules/use-getter-return)

**Comment by ematipico** [comment]
Great write up! Let's promote them!

I think we should recommend all the rules under the a11y category (some of them aren't)

**Comment by ematipico** [comment]
Implemented in https://github.com/biomejs/biome/pull/287
